BlockSettleUILib/Trading/RFQTicketXBT.cpp

```diff
@@ -949,10 +949,25 @@ void RFQTicketXBT::sendRFQ(const std::string &id)
                            the cc side of the tx since only the xbt side covers network fees.
                            */
                            , 1); 
-                        rfq->coinTxInput = BinaryData::fromString(txReq.serializeState().SerializeAsString()).toHexStr();
-                        auto reservationToken = utxoReservationManager_->makeNewReservation(
-                           txReq.getInputs(nullptr), rfq->requestId);
-                        submitRFQCb_(id, *rfq, std::move(reservationToken));
+
+                        auto resolveCB = [this, id, rfq]
+                           (bs::error::ErrorCode result, const Codec_SignerState::SignerState &state)
+                        {
+                           if (result != bs::error::ErrorCode::NoError) {
+                              std::stringstream ss;
+                              ss << "failed to resolve CC half signer with error code: " << (int)result;
+                              throw std::runtime_error(ss.str());
```

Comment:
This can be a one-liner like `throw std::runtime_error("fixed text" + std::to_string((int)result));`